lib/pseudolocalization.rb

```diff
@@ -23,8 +26,28 @@ def respond_to_missing?(name, include_private = false)
       end
 
       def translate(locale, key, options)
+        return original_backend.translate(locale, key, options) if key_ignored?(key)
+
         ::Pseudolocalization::I18n::Pseudolocalizer.pseudolocalize(original_backend.translate(locale, key, options))
       end
+
+      private
+
+      def key_ignored?(key)
+        return false unless ignores
+
+        ignores.any? do |ignore|
+          case ignore
+          when Regexp
+            key.to_s.match(ignore)
+          when String
+            File.fnmatch(ignore, key.to_s)
+          else
+            Rails.logger.tagged('Pseudolocalization I18n').error 'Ignore type unsupported. Expects an array of (mixed) Regexp or Strings.'
```

Comment:
We could also be silent, up to you

Reply:
```suggestion
            Rails.logger.tagged('Pseudolocalization I18n').error('Ignore type unsupported. Expects an array of (mixed) Regexp or Strings.')
```

I think it's worth reporting a failed `ignore` somehow so this seems reasonable.